WebRoot/js/zimbraMail/calendar/view/ZmApptQuickAddDialog.js

```diff
@@ -222,51 +178,29 @@ function() {
 
 ZmApptQuickAddDialog.prototype._setFocusToSubjectFeild =
 function(){
-    this._tabGroup.setFocusMember(this._subjectField);
+	this._tabGroup.setFocusMember(this._subjectField);
 };
 
 ZmApptQuickAddDialog.prototype.popup =
 function(loc) {
 	ZmQuickAddDialog.prototype.popup.call(this, loc);
-    this._fbCache.clearCache();
+	//this._fbCache.clearCache();
```

Comment:
should we remove the commented code or if we might need it later can we add a comment about the line?

Reply:
removed